### DISCUSSION TITLE
Висит на получении глав

### DISCUSSION THREAD

**Discussion by FRcryptofans** [lirix360/ReadmangaGrabber#76] [Q&A] [answered]
Как вы вообще с ней работаете? У меня не выходит даже главы получить.

Скачал. распаковал, залогинился на readmanga сайт, скачал куки для сайта через Get cookie, положил в корень папки назвав readmanga.txt, запустил index.html  и выбил ссылку на мангу, нажал получить главы и ничего. Крутится может больше 40 минут.
Даже по примеру и картинки указал мангу https://readmanga.live/marsh_smerti_pod_rapsodiiu_parallelnogo_mira__A5327 думал может ту что я качаю закрыта, но нет даже у нее не получается отобразить список глав, чтобы скачать. Перепровобовал все 4 сайта и кучи манги, все зависает на получении глав.

**Comment by lirix360** [comment, marked as answer]
Для использования утилиты надо качать не исходный код, а скомпилированную версию из [раздела релизов](https://github.com/lirix360/ReadmangaGrabber/releases/latest) или собирать из исходников самостоятельно.
